### PR TITLE
use directly `Element.prototype.matches`

### LINKED ISSUE #90: ship as ES6 module?
`src/inert.js` could be used as ES6 module if it wasn't for https://github.com/necolas/dom-matches/issues/3
Would you consider implementing that polyfill instead of having it as a dev dependency?

### PR DESCRIPTION
**Pull request by valdrinkoshi**
Fixes #90
Also, fixed build by downgrading `rollup-plugin-commonjs` to ^8.0.0.

### CODE REVIEW

**Comment by robdodson** [comment]
Hey Valdrin, I think similar to focus-visible, we don't want to bundle polyfills with the polyfill itself. This is based on W3C's attempt at polyfill guidance outlined here https://github.com/WICG/focus-visible#dependencies. Instead we ship ES5 and if folks need a feature polyfill we encourage them to use polyfill.io

**Comment by valdrinkoshi** [comment]
Would that mean that we'd only use `element.matches` here instead of having an utility function `matches`?

**Comment by robdodson** [comment]
Correct, we would just use Element.matches. And we would tell folks who need to polyfill it to use polyfill.io. This seems to have worked ok for focus-visible.

**Comment by valdrinkoshi** [comment]
Done, updated README.md too 👌

File: src/inert.js [hunk 1]
```diff
@@ -3,9 +3,18 @@
...
 
-import matches from 'dom-matches';
-
 (function(document) {
+/**
+ * @param {Node} elem
+ * @param {string} selector
+ * @return {boolean}
+ */
+function matches(elem, selector) {
```
[the remark this answers is not in the document]
Comment (valdrinkoshi) [reply src/inert.js]: sure, was just afraid it was used for non-Element nodes but it doesn't look like it

**Pull request merged**